src/components/NavBar/Navbar.css

```diff
@@ -0,0 +1,41 @@
+/* Navbar.css */
+
+/* Background color and font styling */
+.custom-navbar {
+    background-color: #2D2A55; /* Dark purple background */
```

Comment:
I think some of these comments are excessive. What if we change these colors? We have to remember to change the comments to keep them up-to-date too.

I think leaving comments over the selector is OK if they provide context on what they apply to, so feel free to leave those.